bonfire/Modules/Guides/Helpers/toc_helper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+if (! function_exists('load_recursive_guide')) {
+	/**
+	 * Generates a list of guides when recursively search all subfolders
+	 *
+	 * @return mixed
+	 */
+	function load_recursive_guide($alias, $page, $folder)
+	{
+		// Name of folder
+		$string = '<h3>' . esc(ucwords(trim(Bonfire\Modules\Guides\Libraries\GuideCollection::formatPage($folder), ' /'))) . '</h3>
+					<ul class="list-unstyled px-4">';
+
+		 foreach($page as $sub_folder => $row) {
+			 if( is_numeric($sub_folder) ) {
+				 // Is file
+				 $string .= '<li>
+								<a href="/' . ADMIN_AREA . '/guides/' . $alias . '-' . trim($folder, ' /')  . '/' . trim($row, ' /') .'">
```

Comment:
In order not to make changes to the routes, I thought of separating the folder tree with the "-" character and passing the whole segment to the controller